src/protagonist/API.Tests/Integration/ModifyAssetWithoutOldDeliveryChannelEmulationTests.cs

```diff
@@ -0,0 +1,89 @@
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using DLCS.Core.Types;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Test.Helpers.Integration;
+using Test.Helpers.Integration.Infrastructure;
+
+namespace API.Tests.Integration;
+
+[Trait("Category", "Integration")]
+public class ModifyAssetWithoutOldDeliveryChannelEmulationTests : IClassFixture<ProtagonistAppFactory<Startup>>
```

Comment:
Since this flag will be shortlived (it's just for cutover or wellcome deployment) it might make sense to make this class `ModifyAssetWithOldDeliveryChannelEmulationTests` so that we can just drop the entire class when the flag is removed, rather than deleting tests from main `ModifyAssetTests` and moving these over.